Sources/MisticaCommon/Colors/ColorToolkit+UIColor.swift

```diff
@@ -653,7 +658,7 @@ public extension UIColor {
 public extension BrandStyle {
     var preferredStatusBarStyle: UIStatusBarStyle {
         switch self {
-        case .movistar, .vivo, .o2, .o2New, .blau, .custom, .vivoNew, .telefonica, .tu:
+        case .movistar, .vivo, .o2, .o2New, .blau, .custom, .vivoNew, .telefonica:
```

Comment:
Please do not erase `tu` brand:
```suggestion
        case .movistar, .vivo, .o2, .o2New, .blau, .custom, .vivoNew, .telefonica, .tu:
```

Reply:
This PR was automatically generated by tokens automation 😬